include/host1x.h

```diff
@@ -161,12 +161,12 @@ int host1x_overlay_set(struct host1x_overlay *overlay,
 
 #define HOST1X_BO_CREATE(host1x, size, flags)				\
 ({									\
-	struct host1x_bo *bo = host1x_bo_create(host1x, size, flags);	\
-	if (!bo)							\
+	struct host1x_bo *_bo_ = host1x_bo_create(host1x, size, flags);	\
+	if (!_bo_)							\
 		fprintf(stderr,						\
 			"ERROR: %s:%d: host1x_bo_create() failed\n",	\
 			__FILE__, __LINE__);				\
-	bo;								\
+	_bo_;								\
 })
```

Comment:
Hmpf, this is a bit nasty IMO (having to worry about outer scopes just to avoid using an inline function, underscore identifiers being reserved by the compiler)... Maybe sticking the logic inside a static-inline helper is nicer? Something like

```c
static inline struct host1x_bo *host1x_bo_create_helper(struct host1x *host1x, size_t size, int flags, const char *file, int line)
{
	struct host1x_bo *bo = host1x_bo_create(host1x, size, flags);
	if (!bo)
		fprintf(stderr,
 			"ERROR: %s:%d: host1x_bo_create() failed\n",
			file, line);
	return bo;
}
#define HOST1X_BO_CREATE(host1x, size, flags) host1x_bo_create_helper(host1x, size, flags, __FILE__, __LINE__)
```
instead?